lix/client/package.json

```diff
@@ -23,6 +23,7 @@
 	},
 	"dependencies": {
 		"@inlang-git/fs": "*",
+		"@inlang/app": "*",
```

Comment:
@janfjohannes why does lix depent on inlang/app? 

If we want to share reactive primitives, lets extract the reactive primitives and adapters from inlang/app to lix/reactive-primitives or something. 

The logical education is that "inlang is a lix app". Not the other way around "lix is an inlang app"

Reply:
Related to https://github.com/inlang/inlang/issues/1290